doc/_static/mpl.css

```diff
@@ -683,20 +683,64 @@ figcaption {
     outline-color: #003c63;
 }
 
-.flex-container {
-    /* display part copy-pasted from: https://css-tricks.com/using-flexbox/ */
-    display: -webkit-box;      /* OLD - iOS 6-, Safari 3.1-6 */
-    display: -moz-box;         /* OLD - Firefox 19- (buggy but mostly works) */
-    display: -ms-flexbox;      /* TWEENER - IE 10 */
-    display: -webkit-flex;     /* NEW - Chrome */
-    display: flex;             /* NEW, Spec - Opera 12.1, Firefox 20+ */
-
-    flex-wrap: wrap;
-    flex-direction: row;
-    align-content: center;
-    justify-content: center;
-}
-
-.flex-item {
-    margin: 0px;
+
+div.responsive_banner {
+    /* Horizontally centered */
+    display: block;
+    margin: auto;
+
+    max-height: 139px; /* at most 1 x 1:1 subfig height */
+    width=100%;
+    overflow: auto; /* to follow the height of the subfig floats */
+}
+
+div.responsive_screenshots {
+    /* Horizontally centered */
+    display: block;
+    margin: auto;
+
+    /* Do not go beyond 1:1 scale*/
+    max-width: 648px; /* 4 * 1:1 subfig width */
+    max-height: 139px; /* 1 * 1:1 subfig height */
+
+    /* Possible downscaling */
+    width=auto;
+    height: 100%;
+}
+
+div.responsive_subfig{
+    float: left;
+    width: 25%;  /* 4 subfigs in a row */
```

Comment:
I believe that magic `24.99999%` variable is necessary for some browsers. See http://www.otsukare.info/2016/05/25/width-rounding-in-css